gateway/engine/beans/pom.xml

```diff
@@ -12,6 +12,10 @@
 
   <dependencies>
     <dependency>
+      <groupId>net.openhft</groupId>
+      <artifactId>zero-allocation-hashing</artifactId>
+    </dependency>
+    <dependency>
```

Comment:
Thanks, Nick! Appreciated.

> On 26 Apr 2016, at 10:41, Nick Cross <notifications@github.com> wrote:
> 
> In gateway/engine/beans/pom.xml <https://github.com/apiman/apiman/pull/460#discussion_r61056812>:
> 
> > @@ -12,6 +12,10 @@
> >  
> >    <dependencies>
> >      <dependency>
> > +      <groupId>net.openhft</groupId>
> > +      <artifactId>zero-allocation-hashing</artifactId>
> > +    </dependency>
> > +    <dependency>
> @msavy <https://github.com/msavy> @EricWittmann <https://github.com/EricWittmann> It appears to be Apache 2.0 licensed and a Maven build so its fine. Thanks for asking.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/apiman/apiman/pull/460/files/37b99699b175c1b33ab65e353ebef9a3e753604b#r61056812>